### PR TITLE
 BPTL Dashboard: Added endpoints for package receipt & bptl metrics

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
This PR addresses following feature:
-> Added POST endpoint for package receipt
-> Added GET endpoint for bptl metrics

Checklist:
- [x] Code cleanup
- [x] Check for ES Lint warnings
- [ ] Verify test cases
- [x] Check for GIT conflicts
- [ ] Verified unit tests pass with current changes
- [ ] Any dependencies or modules added
- [ ] Attach PoC
- [x] Notes added

**Notes:**

_**http://{URL}/biospecimen?api=storeReceipt**_ stores package receipts information for participants
_**http://{URL}/biospecimen?api=bptlMetrics**_ returns BPTL metrics

### CODE REVIEW

File: utils/firestore.js [hunk 1]
```diff
@@ -1406,15 +1425,77 @@ const shipKits = async (data) => {
...
+         }
+    catch(error){
+        return new Error(error);
+    }
+}
+
+const kitStatusCounterVariation = async (currentkitStatus, prevKitStatus) => {
+    try {
+        await db.collection("bptlMetrics").doc('--metrics--').update({ 
+            [currentkitStatus]: increment
```
Comment (bhaumik55231) [inline utils/firestore.js:1480]: where are increment and decrement defined?

File: utils/firestore.js [hunk 2]
```diff
@@ -1406,15 +1425,77 @@ const shipKits = async (data) => {
...
+        })
+        await db.collection("bptlMetrics").doc('--metrics--').update({ 
+            [prevKitStatus]: decrement
```
Comment (bhaumik55231) [inline utils/firestore.js:1483]: where are increment and decrement defined?

File: utils/firestore.js (hunk 1 above)
Comment (abhinavjonnada82) [reply utils/firestore.js:1480]: https://github.com/episphere/connectFaas/pull/94/files/71a08e3a8dcbca30f20c8c072c3fe56fc80b2167#diff-be91e66430aa4dd2f61668d1bd17d3cb449f57947e13bfa2f6bc087919967282R11

File: utils/firestore.js (hunk 2 above)
Comment (abhinavjonnada82) [reply utils/firestore.js:1483]: https://github.com/episphere/connectFaas/pull/94/files/71a08e3a8dcbca30f20c8c072c3fe56fc80b2167#diff-be91e66430aa4dd2f61668d1bd17d3cb449f57947e13bfa2f6bc087919967282R11

**Pull request merged**